docs-sandbox/docs/08-deploy/index.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: Deploy
+slug: /deploy
+version: '3.0'
+section: docs
+category: developers
+keywords:
```

Comment:
Should we seed all pages now with a handful of keywords that we think are core to each 1st level of the docs hierarchy? We can tweak latter on of course.
And if so, then copy these by default into higher level dirs and pages?

example here:
```markdown
---
slug: /deploy
kewords:
  - deployment
  - devops
  - launch
  - CI
  - CD
  - .........
---
```
Another page in `/develop`:

```markdown
---
slug: /deploy/cloud-hosting
kewords:
  - deployment
  - devops
  - launch
  - CI
  - CD
  - AWS
  - GCS
  - GCP
  - .........
---
```

Reply:
TO the extent possible, yes.